From 0c605e3e57ad83bafda215605ed9b73af77d4e3d Mon Sep 17 00:00:00 2001
From: Tim Snyder <snyder.tim@gmail.com>
Date: Tue, 30 Jan 2024 22:02:15 +0000
Subject: [PATCH 1/1] rm project-default CXX std

use what the toolchain defaults to as with all other conda-forge packages
---
 CMakeLists.txt | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b7815aef..ac33e8d2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -273,12 +273,6 @@ if (NOT CMAKE_BUILD_TYPE)
        FORCE)
 endif (NOT CMAKE_BUILD_TYPE)
 
-set (CMAKE_CXX_STANDARD 98 CACHE STRING
-     "C++ standard to build all targets. Supported values are 98, 11, and 14.")
-set (CMAKE_CXX_STANDARD_REQUIRED ON CACHE BOOL
-     "The with CMAKE_CXX_STANDARD selected C++ standard is a requirement.")
-mark_as_advanced (CMAKE_CXX_STANDARD_REQUIRED)
-
 if (NOT (WIN32 OR CYGWIN))
   option (BUILD_SHARED_LIBS "Build shared libraries." ON)
 else (NOT (WIN32 OR CYGWIN))
-- 
2.17.1

